code/game/objects/items/holy_weapons.dm

```diff
@@ -309,6 +310,7 @@
 /obj/item/nullrod/claymore
 	icon_state = "claymore_gold"
 	inhand_icon_state = "claymore_gold"
+	worn_icon = 'icons/mob/clothing/back.dmi'
```

Comment:
handled by build_worn_icon() already

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.